topics/metagenomics/tutorials/general-tutorial/workflows/amplicon-job.yml

```diff
@@ -1,19 +1,19 @@
 ---
-silva:
+'SILVA Reference':
   class: File
-  location: https://zenodo.org/record/815875/files/silva.v4.fasta
+  location: https://zenodo.org/api/files/0e062e89-e6ce-463b-a8cb-0151e5aabc69/silva.v4.fasta
   filetype: fasta
-anguil_soil_sample:
+'Anguil Soil Sample':
   class: File
-  location: https://zenodo.org/record/815875/files/SRR651839_anguil.fasta
+  location: https://zenodo.org/api/files/0e062e89-e6ce-463b-a8cb-0151e5aabc69/SRR651839_anguil.fasta
   filetype: fasta
-pampa_soil_sample:
+'Pampa Soil Sample':
   class: File
-  location: https://zenodo.org/record/815875/files/SRR531818_pampa.fasta
-trainset_alignment:
+  location: https://zenodo.org/api/files/0e062e89-e6ce-463b-a8cb-0151e5aabc69/SRR531818_pampa.fasta
+'Trainset PDS alignment':
   class: File
-  location: https://zenodo.org/record/815875/files/trainset16_022016.pds.fasta
-trainset_taxonomy:
+  location: https://zenodo.org/api/files/0e062e89-e6ce-463b-a8cb-0151e5aabc69/trainset16_022016.pds.fasta
```

Comment:
@cat-bro do you know which URL type is the recommended one? You changed it here to be a UID like. But below you are using `https://zenodo.org/record/1051552/files/HEK_SILAC-K6R6_ST905_part.mzml` the record ones again.

I'm wondering if one more stable than the other?

Reply:
@bgruening I don't know which URL is recommended.  The `files` urls are used in the data libraries.  There was probably no need to update the links in this case.

Reply:
Do you want to redo the changes to this file then? Otherwise :+1: from me! Thanks.

Reply:
done :--)